src/test/resources/pages/studentFeedbackSubmitPageGracePeriod.html

```diff
@@ -66,7 +66,7 @@
               </label>
               <div class="col-sm-10">
                 <p class="form-control-static">
-                  ${datetime.now}
+                  ${datetime.now} UTC+0800
```

Comment:
@HirdayGupta One of my [PR's  Travis build](https://travis-ci.org/TEAMMATES/teammates/builds/255814506#L2059) failed due to `${datetime.now} UTC+0800`

Suggestions were to change `${datetime.now} UTC+0800` to `Thu, 20 Jul 2017, 09:21 PM UTC+0800`